apps/www/registry/new-york/ui/timeline.tsx

```diff
@@ -0,0 +1,75 @@
+"use client"
+
+import * as React from "react"
+
+import { cn } from "@/lib/utils"
+
+const Timeline = React.forwardRef<
+  HTMLDivElement,
+  React.HTMLAttributes<HTMLDivElement>
+>(({ className, ...props }, ref) => (
+  <div ref={ref} className={cn("space-y-4", className)} {...props} />
+))
```

Comment:
Have you considered adding a `orientation ` prop to support both row and column layouts? 

The current implementation seems to only handle vertical (column) layout. Adding a simple direction prop would make the component more flexible for different use cases.

Reply:
Yeah, You are right. I shall add the direction props to support both row and column layouts.